jobs/JGLOBAL_ATMOS_POST

```diff
@@ -117,9 +124,9 @@ export SLEEP_INT=5
 ###############################################################
 # Run relevant exglobal script
 
-$SCRgfs/ex${RUN}_atmos_nceppost.sh
+"${SCRgfs}/ex${RUN}_atmos_post.sh"
```

Comment:
Are the `exgfs_atmos_post.sh` and `exgdas_atmos_post.sh` that different?
Just curious here.

Reply:
Not too different. Comparing more carefully and merging them is part of the next phase. This PR is supposed to just be renaming the scripts (and then all the linter complaints).